draft-steele-cose-merkle-tree-proofs.md

```diff
@@ -306,7 +297,7 @@ See {{-certificate-transparency-v2}}, 2.1.3.1. Generating an Inclusion Proof.
 The cbor representation of the inclusion proof is:
 
 ~~~~ cddl
-inclusion-proof = #6.1234([
+inclusion-proof = #TBD_2([
```

Comment:
any change suggestions on this CDDL very welcome.

Reply:
good enough for now